modyn/supervisor/internal/pipeline_executor/models.py

```diff
@@ -273,6 +273,7 @@ def df(self) -> pd.DataFrame:
                     self.eval_request.training_id,
                     self.eval_request.id_model,
                     self.eval_request.most_recent_model,
```

Comment:
Most recent model is gone from the evaluator. This should crash, shouldn't it?

Reply:
I think it's only used in the analytics tool, so I didn't notice so far.